extensions/amp-consent/0.1/consent-config.js

```diff
@@ -245,19 +244,23 @@ export class ConsentConfig {
   }
 
   /**
-   * Validate if story consent then no promptUiSrc
+   * Validate if story consent then no promptUISrc
    * @param {!JsonObject} config
    * @return {!JsonObject}
    */
   checkStoryConsent_(config) {
-    if (childElementByTag(this.element_, AMP_STORY_CONSENT_TAG)) {
-      userAssert(
-        !config['promptUISrc'],
-        '%s: `promptUiSrc` cannot be specified while using %s.',
-        TAG,
-        AMP_STORY_CONSENT_TAG
-      );
-    }
+    // TODO(wg-stories):
+    // We verify that configurations with an <amp-story-consent> descendant have
+    // no promptUISrc, but we don't verify that the element is present in
+    // Stories with <amp-consent>. Thus, Stories could be configured using a
+    // promptUISrc. This is likely not what we want for UX, so wg-stories
+    // should provide a migration path for these documents if they exist.
```

Comment:
If there is an issue with the Stories consent component, please verify that it does exist, and file a ticket with actionable context. Adding a TODO in a file that the Stories working group don't maintain will never resolve the issue.

Reply:
I'd be happy to start an issue with more context, but I lack the context. Please see [my other comment.](https://github.com/ampproject/amphtml/pull/35571#discussion_r687268266)